qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -75,17 +75,17 @@
             chardev_abstract_vs1 = 'on'
             variants:
                 - tight_on:
-                     chardev_tight_vs1 = 'on'
+                    chardev_tight_vs1 = 'on'
                 - tight_off:
-                     chardev_tight_vs1 = 'off'
+                    chardev_tight_vs1 = 'off'
     variants:
         - unix_socket:
         - tcp_socket:
             chardev_backend_vs1 = tcp_socket
         - pty:
             chardev_backend_vs1 = pty
     variants:
-        - @default:                                                              
+        - @default:
```

Comment:
Update these code due to it will introduce some CI check failure.

https://github.com/autotest/tp-qemu/actions/runs/8044679165/job/21968654468?pr=3965